BUILDING.md

```diff
@@ -24,6 +24,10 @@ The hostname and port of the instance can be changed with the following user def
 * `aem.host` and `aem.port` for the author instance.
 * `aem.publish.host` and `aem.publish.port` for the publish instance.
 
+### AEM as a Cloud Service SDK
+
+When compiling and deploying to AEM as a Cloud Service SDK, you can use the `cloud` profile (in conjunction with previously documented profiles) to generate `cloud`-ready artifacts (with components located in `/libs` instead of `/apps`). To allow recompilation of the HTL scripts, you should disable `aem-precompiled-scripts` bundle.
+
```

Comment:
Probably worth to mention that any other Core WCM bundle versions (the release ones) should be disabled due to https://issues.apache.org/jira/browse/FELIX-6365. Also the new profile `install-bundle` should be mentioned

Reply:
I did not come across this issue when working with the latest snapshots:
![image](https://user-images.githubusercontent.com/3670963/101800496-fffc2a00-3b15-11eb-86dd-6ce5e9c4bcaa.png)

Reply:
I am talking about the same BSN in two versions. Your screenshot shows two BSNs in two versions!
How did you deploy the SNAPSHOT version bundle then?

Reply:
```
[INFO] --- maven-sling-plugin:2.1.0:install (default-cli) @ core.wcm.components.core ---
[INFO] Installing Bundle com.adobe.cq.core.wcm.components.core(/Users/vlad/dev/aem/cq/modules/sample/aem-core-wcm-components/bundles/core/target/core.wcm.components.core-2.13.1-SNAPSHOT.jar) to http://localhost:4502/system/console/bundles via POST
```
That was deployed on top of `2.13.0`. The amp bundle was not deployed, it's there just as an example.

Reply:
Ok, the order of deployment is relevant: 

1. If you deploy the first time via `autoInstallSinglePackage` the deployment of the SNAPSHOT bundle goes via the OSGi installer, which will deploy the bundle in parallel(!)
2. If you deploy the bundle again via `autoInstallPackage` (which goes via the Felix Web Console ReST API) you trigger the bug

Reply:
I see what you mean:

Reply:
Updated docs in e330129

Reply:
Thanks, LGTM now